scripts/breadcrumbs.js

```diff
@@ -0,0 +1,68 @@
+import { fetchPageIndex } from './pages.js';
+import {
+  a, div, li, span, ul,
+} from './dom-helpers.js';
+
+export default async function buildBreadcrumbs() {
+  const { data } = await fetchPageIndex();
+
+  const { pathname } = window.location;
+  const pathParts = pathname.split('/').filter((part) => part);
+
+  let currentPath = '';
+  const breadcrumbItems = [];
+
+  // Helper function to find all page names by path
+  function getPageNamesByPath(path) {
+    return data.filter((page) => page.path === path).map((page) => page['page name']);
+  }
+
+  const homeLink = a({ href: '/' }, 'Home');
+  const homeCrumb = li(homeLink);
+  const crumbList = ul({ class: 'breadcrumbs' }, homeCrumb);
+  const breadcrumbContainer = div({ class: 'section' }, crumbList);
+
```

Comment:
You could slim this down, I think it's easier to read this way - but either is fine. 

```
  const breadcrumbContainer = div({ class: 'section' },
    ul({ class: 'breadcrumbs' }, 
      li(a({ href: '/' }, 'Home')),
    );
```

Reply:
Looks like you need homeCrumb as an object - just caught that below